src/PanelContentsWithSubpanel/PanelContentsWithSubpanel.tsx

```diff
@@ -1,64 +0,0 @@
-import { ReactNode, useEffect, useRef, useState } from "react";
-import panelContentsWrapperStyles from "./PanelContentsWithSubpanel.module.scss";
-import { ButtonXTopRight } from "../couldBeSharedComponents/buttons/ButtonXTopRight";
-import { useCloseSubpanelWhenParentPanelCloses } from "./useCloseSubpanelWhenParentPanelCloses";
-
-export const PanelContentsWithSubpanel = ({
-  isLeftPanelOpen,
-  isSubpanelOpen = false,
-  mainPanelContent,
-  setIsSubpanelOpen = () => {},
-  subpanelClassName = undefined,
-  subpanelContent,
-}: {
-  isLeftPanelOpen: boolean;
-  isSubpanelOpen?: boolean;
-  mainPanelContent: ReactNode;
-  setIsSubpanelOpen?: (arg: boolean) => unknown;
-  subpanelClassName?: string;
-  subpanelContent: ReactNode;
-}) => {
-  const [subpanelWidthFromCss, setSubpanelWidthFromCss] = useState<string>();
-
-  const subpanelRef = useRef<HTMLDivElement>(null);
-
-  const closeSubpanel = () => {
-    setIsSubpanelOpen(false);
-  };
-
-  useCloseSubpanelWhenParentPanelCloses({
-    isLeftPanelOpen,
-    setIsSubpanelOpen,
-  });
-
-  useEffect(function getSubpanelWidthValueFromCss() {
-    setSubpanelWidthFromCss(`${subpanelRef.current?.offsetWidth}px`);
```

Comment:
This is my fun blunder. At one point we had a `width` prop for convenience, but we decided to go pure css instead of having styling props, so I guess this thinking is leftover from that, or maybe it didnt make sense then either, but I didnt think until I found a bug to remove this unnecessary JS CSS micromanagement.